cid-redirects.json

```
@@ -2011,7 +2009,7 @@
   "/cid/5191": "/docs/integrations/containers-orchestration/vmware-legacy",
   "/cid/5192": "/docs/integrations/amazon-aws/cloudfront",
   "/cid/5193": "/docs/integrations/security-threat-detection/cisco-asa",
-  "/cid/5194": "/docs/integrations/sumo-apps/data-volume-legacy",
+  "/cid/5194": "/docs/integrations/sumo-apps/data-volume-legacy :/docs/integrations/sumo-apps/data-volume",
```

You need to replace the URL, not add to it

```suggestion
  "/cid/5194": "/docs/integrations/sumo-apps/data-volume",
```